Makefile

```diff
@@ -139,15 +141,26 @@ ui:
 	$(OPEN) https://$(MASTER_LB_IP)
 
 .PHONY: install
-install: check-dcos
+install: check-dcos add-stubs
 	@echo "Installing Mesosphere Kubernetes Engine..."
 	$(DCOS_CMD) package install --yes kubernetes --package-version="$(KUBERNETES_FRAMEWORK_VERSION)"
 	@echo "Waiting for Mesosphere Kubernetes Engine to be up..."
 	@while [[ ! $$($(DCOS_CMD) kubernetes manager plan show deploy 2> /dev/null | head -n1 | grep COMPLETE ) ]]; do \
 		sleep 1; \
 	done
 	@echo "Creating a Kubernetes cluster..."
-	$(DCOS_CMD) kubernetes cluster create --yes --options="$(PATH_TO_PACKAGE_OPTIONS)"
+	$(DCOS_CMD) kubernetes cluster create --yes --options="$(PATH_TO_PACKAGE_OPTIONS)" --package-version="$(KUBERNETES_FRAMEWORK_VERSION)"
+
+.PHONY: add-stubs
+add-stubs:
+ifdef KUBERNETES_STUB_URL
+	@echo "Adding 'kubernetes' stub"
```

Comment:
Should we not try to remove a previous stub first?

Reply:
It's being removed as part of the `uninstall`. I don't really feel strongly either way.

Reply:
ahh i thought you wanted to use this to test stubs.